forum/frontend/src/components/LoginSuccess.js

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import axios from 'axios';
+
+const LoginSuccess = () => {
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    const handleGoogleLogin = async () => {
+      const urlParams = new URLSearchParams(window.location.search);
+      const code = urlParams.get('code');
+
+      if (code) {
+        try {
+          const response = await axios.post('http://127.0.0.1:8000/api/token/oauth/', {
```

Comment:
Replace hardcoded API URLs with environment variables for better flexibility in different environments (development, production).